components/datepicker/src/auro-datepicker.js

```diff
@@ -861,6 +858,11 @@ export class AuroDatePicker extends LitElement {
     }
   }
 
+  handleSlotToSlot(event) {
```

Comment:
Need JSDoc comments about this function.

Reply:
added